online-fishing-store/src/app/pages/main-page/sort-panel/sort-panel.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { SortButtonDirection } from 'src/app/enums/sort-button-direction-enum';
+import { SortButtonLabel } from 'src/app/enums/sort-button-label-enum';
+import { SortButtonValue } from 'src/app/enums/sort-button-value-enum';
+import { SortButton } from 'src/app/interfaces/sort-button';
+
+@Component({
+  selector: 'app-sortPanel',
+  templateUrl: './sort-panel.component.html',
+  styleUrls: ['./sort-panel.component.scss'],
+})
+export class SortPanelComponent {
+  @Input() public sortButtons: SortButton[];
+  @Output() public sorting: EventEmitter<SortButton> =
+    new EventEmitter<SortButton>();
+
+  public isSortByPriceASC: boolean = true;
+  public selectedButton: string = SortButtonValue.RATING;
```

Comment:
By default you don't have selected button. Because when you load a page you don't have to sort anything